incuna-sass/helpers/_link.sass

```diff
@@ -1,18 +1,19 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
-        text-decoration: underline
-        &:hover
-            text-decoration: none
     @if $type = reverse
         text-decoration: none
         &:hover
             text-decoration: underline
-    @if $type = blank
-        text-decoration: none
-        &:hover
+    @else
```

Comment:
Or can you do
```
@if type = blah
@else @if type = ugh
@else
```
?
It's generally better to avoid nesting of if/else if possible.